themes/bootstrap3/less/components/ui/confirm-menu.less

```diff
@@ -0,0 +1,57 @@
+.confirm-menu {
```

Comment:
For some reason, the `lessToSass` task isn't creating an SCSS version of this component. Is something broken there? Is it possible that other SCSS templates are out of sync with their LESS equivalents?